testsuite/run_forge.sh

```diff
@@ -93,7 +97,19 @@ set_image_tag() {
             if [ "$?" -eq 0 ]; then
                 echo "Image tag exists. Using tag: ${IMAGE_TAG_DEFAULT}"
                 IMAGE_TAG=$IMAGE_TAG_DEFAULT
-                return 0
+                break
+            fi
+        done
+        # get the OLD_IMAGE_TAG
+        for i in $(seq 1 $commit_threshold); do
```

Comment:
Hmm ..this doesn't look right to me. It might happen that `IMAGE_TAG` and `OLD_IMAGE_TAG` are the same which is not what we want right?

Reply:
Not quite. We're iterating on git revs from `git rev-parse $IMAGE_TAG~$i` from `i = 1` onwards, so we're guaranteed an older revision

Reply:
Ah, I see - missed that part.